modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -221,54 +224,51 @@ image::monitoring-dashboards-meta.png[]
 
 == Server Group and Cluster Dashboards
 
-=== Access the Server Group or Cluster Dashboard
+The server group or cluster dashboard displays three aggregated metrics:
 
-To access the dashboard of a cluster or a server group on-premises:
+* CPU usage, as a percentage of the capacity
+* Memory usage, in MB
+* Heap total, in MB
 
-. Select a cluster or server group in the *Servers* page to open the side panel.
-. Click the *View Dashboard* button. +
-The dashboard for the server group or cluster displays. 
+The aggregated metrics of the selected servers appear as a single plot line on each graph.
 
-=== Navigating the Dashboard
+=== Access the Server Group or Cluster Dashboard
 
-The dashboard displays three separate metrics:
+To access the dashboard of a server group or cluster for on-premises deployment:
 
-* CPU usage, as a percentage of the capacity
-* Memory usage, in MB
-* Heap total, in MB
+. In Runtime Manager, click *Servers*.
+. Click the cluster or server group name to display the dashboard.
+. Select one or more servers or applications from the list to display aggregated metrics in the details pane.
 
-image::monitoring-dashboards-5cd3c.png[]
+=== Navigating the Group or Cluster Dashboard
 
-You can view graphs for specific time periods by selecting the desired time interval in the top-right corner.
+// image::monitoring-dashboards-5cd3c.png[]
 
-image::monitoring-dashboards-cbd92.png[]
+You can view graphs for specific periods by selecting the time interval in the top-right corner.
 
-The aggregated metrics of the entire set of servers included will be charted as a single plot line on the main set of graphs.
+image::monitoring-dashboards-cbd92.png[]
 
-You can break down this information into either individual servers or individual applications, note the two tabs that allow you to pick a perspective:
+You can display this information for either servers or applications by clicking the appropriate tab:
 
 image::monitoring-dashboards-6fe9f.png[]
 
-Then, select one or many servers or applications, and a side-panel will display on the right with this data.
+From this dashboard, you can click a <<Access the Server Monitoring Dashboard,server>> or <<Access the Application Monitoring Dashboard,application>> name to access the dashboard for that item.
 
-From this menu, you can also click on an individual server name to be taken to the dashboard page for <<The Dashboard for a Server, that individual server>>.
 
-[TIP]
-Set up xref:alerts-on-runtime-manager.adoc#conditions-on-mule-servers[Alerts] to notify you automatically whenever any of these metrics hits a critical value for a sustained period of time.
+== Alerts
```

Comment:
Does this and the following topic really deserve topic status? Is this something we can add to the intro text somewhere earlier?